### DISCUSSION TITLE
.Net Core 1.0.0 RC2 Support

### DISCUSSION THREAD

**Discussion by JsonRoob** [Dotnet-Boxed/Templates#98] [Q&A]

**Comment by RehanSaeed** [comment]
Almost there, you can try doing a git clone and testing the master branch if you'd like. I'm planning on adding a framework (.Net Core or Full .Net) feature selector and maybe a runtime (portable, ubuntu. windows10, windows 8, windows7 etc.) feature selector before releasing.

**Comment by timematcher** [comment]
Great work! Looking forward to these great features soon!

**Comment by Huntk23** [comment]
On your most recent commits a couple hours ago, I attempted creating a MVC6 project and running into a couple issues:
1. SyntaxError: config.json: Unexpected token
   Fixed by removing a comma on line 19 within the CacheProfileSettings option (this is because I only have the CacheProfiles of **Error** and didn't select the rest of the options, Feed, OpenSearchXml, etc.
2. 1>project.json(6,53): error NU1001: The dependency Boilerplate.AspNetCore >= 1.0.0-rc2-final-0001 could not be resolved.
   1>project.json(7,64): error NU1001: The dependency Boilerplate.AspNetCore.TagHelpers >= 1.0.0-rc2-final-0001 could not be resolved.
   Were these sub projects supposed to be included in the project creation?

Last question, you said you are adding a framework feature selector, is this hard to do right now manually for .NET 4.6.1 and avoid using Core?

~~I was wondering if all that needs to be done is change the project.jsons from Microsoft.AspNetCore.\* to Microsoft.AspNet.*?~~

Thanks for the awesome template.

**Comment by RehanSaeed** [comment]
@Huntk23 
1. That's an annoyingly difficult problem to solve. One solution I've been thinking about is a JSON fixer which is run at the end and removes any commas in the wrong place.
2. I've not published the NuGet packages yet.

As for the target framework feature selector, it would modify the `frameworks` section of project.json and also remove the `#if NET461` pre-processor directives if the full .NET framework is selected. Nothing complicated, just a nice to have and fairly simple to implement..

**Comment by RehanSaeed** [comment]
NuGet packages are published.

**Comment by RehanSaeed** [comment]
You can check the release notes [here](https://github.com/RehanSaeed/ASP.NET-MVC-Boilerplate/blob/master/Source/Boilerplate.Vsix/Release%20Notes.txt) for info on what is changing.

**Comment by RehanSaeed** [comment]
Released!
